packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -388,19 +380,9 @@ export default class Editor {
         let contentDiv = this.core.contentDiv;
         let contentChanged = false;
         if (contentDiv.innerHTML != content) {
-            contentDiv.innerHTML = content || '';
+            let range = setHtmlWithSelectionPath(contentDiv, content);
+            this.select(range);

```

Comment:
in the old code, `this.select(range)` was called in the try block -- is it fine to call `this.select(null)`?

Reply:
yes. It's ok to call with null, and it will be ignored.